live/tests/unit/services/mail-generator-test.js

```diff
@@ -0,0 +1,95 @@
+import { expect } from 'chai';
+import { describe, it } from 'mocha';
+import { setupTest } from 'ember-mocha';
+import sinon from 'sinon';
+
+describe('Unit | Service | mail generator', function() {
+  setupTest('service:mail-generator', {});
+
+  // Replace this with your real tests.
+  it('exists', function() {
+    const service = this.subject();
+    expect(service).to.be.ok;
+  });
+
+  it('should have a generateEmail function', function() {
+    // Given
+    const service = this.subject();
+
+    // When
+    expect(service).to.have.property('generateEmail')
+      .and.to.be.a('function');
+  });
+
+  describe('#generateEmail', function() {
+    let service;
+    let clock;
+    const februaryTheFifth = new Date(2017, 1, 5);
+
+    beforeEach(function() {
+      service = this.subject();
+      clock = sinon.useFakeTimers(februaryTheFifth);
+    });
+
+    afterEach(() => {
+      clock.restore();
+    });
+
+    it('when the environment is production', function() {
```

Comment:
Ce it me semble plus juste dans un describe, et ajouter un it pour dire ce que fait le test